### PR TITLE
Support treviso backoffice

### PR DESCRIPTION
**Pull request by lchanmann**
## Ref
- https://nanopay.atlassian.net/browse/NP-3386

## Changes
- Use menuDAO default order i.e. order, label
- Place user.group in administrative section to allow msp-admin to save user

### CODE REVIEW

**Review by jlhughes — APPROVED** [review]

File: src/foam/nanos/auth/User.js [hunk 1]
```diff
@@ -849,7 +849,7 @@ foam.RELATIONSHIP({
   },
   targetProperty: {
     hidden: false,
-    section: 'systemInformation',
+    section: 'administrative',
```
Comment (nanoscott) [inline src/foam/nanos/auth/User.js:852]: @lchanmann .. there is no 'administive' section .. why did you change this? now the group cannot be seen
